lib/p5.js

```diff
@@ -124,9 +124,9 @@ amdclean['constants'] = function (require) {
     _DEFAULT_FILL: '#FFFFFF'
   };
 }({});
-amdclean['core'] = function (require, shim, constants) {
+amdclean['core_core'] = function (require, core_shim, core_constants) {
   'use strict';
-  var constants = constants;
+  var constants = core_constants;
```

Comment:
This is an example of the changed build output when we switch from `require('core')` to `require('core/core')`, and so on -- the only actual impact on the built file is that each module gets a different unique key.